### DISCUSSION TITLE
enqueue=True in loguru doesn't work with eventlet in socketIo

### DISCUSSION THREAD

**Discussion by liuqijie6** [miguelgrinberg/Flask-SocketIO#1929] [Q&A]
socket config
```
socket_io = SocketIO(cors_allowed_origins='*',
                     async_mode="eventlet",  # eventlet  gevent # threading
                     engineio_logger=True,
                     logger=True,
                     message_queue='redis://:1234@xxxx:6380/0',
                     )
```

app.py
```
__all__ = ["app"]
import eventlet
eventlet.monkey_patch()
from __init__ import create_app
from units.extensions import socket_io
from config import CONFIG

app = create_app()

if __name__ == "__main__":
    host, port = CONFIG.get("web_server", "host"), CONFIG.get("web_server", "port")
    # print(app.url_map)
    # app.run(host=host, port=port, threaded=False, debug=False)
    socket_io.run(app, host=host, port=port, debug=True)
```

log config file
![image](https://user-images.githubusercontent.com/52588515/211708926-225a18c4-d885-4b54-939c-898c7633ba0e.png)
**cant use.  no run** 
![image](https://user-images.githubusercontent.com/52588515/211709434-bfacadfb-610f-4614-9d09-e60b74e984d7.png)

![image](https://user-images.githubusercontent.com/52588515/211708882-0fa0521f-28d9-4348-8241-aa73812a8f88.png)
**can use**
![image](https://user-images.githubusercontent.com/52588515/211709232-6a8a1fdf-d061-4089-82d5-e082e097ce40.png)

**I do not know why**

**Comment by miguelgrinberg** [comment]
This should be asked to the eventlet team, but in any case, I suspect this logging library that you are using might be incompatible with eventlet.
